decorators/decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
```

Comment:
- Hey, let's also add the accessor method for the nameable so we can access it like this below:

`attr_accessor :nameable`